nixos/modules/services/system/dbus.nix

```diff
@@ -102,10 +82,12 @@ in
 
     users.groups.messagebus.gid = config.ids.gids.messagebus;
 
-    systemd.packages = [ pkgs.dbus.daemon ];
+    systemd.packages = [
+      pkgs.dbus
+    ];
```

Comment:
Why do we need the regular package rather than daemon here?

Reply:
See the commit message, there is no daemon package.